### PR TITLE
change code

### PR DESCRIPTION
**Pull request by yehuditKritenberger**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -3,20 +3,22 @@ const size=5;
...
+console.log(size * 10);
 
 console.log("hello");
 
 console.log("Michal Bassa");
 
 console.log("conflict");
 
-const array=[1, 2, 3];
+const array=[1, 2, 3, 4, 5, 6];
```
Comment (michalBassa) [inline index.js:14]: add to array?

File: index.js [hunk 2]
```diff
@@ -3,20 +3,22 @@ const size=5;
 
 console.log(size);
 
-console.log(size * 7);
+console.log(size * 10);
```
Comment (michalBassa) [inline index.js]: this is corect!

**Review by michalBassa — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (yehuditKritenberger) [reply index.js:14]: ok

File: index.js (hunk 2 above)
Comment (yehuditKritenberger) [reply index.js]: change to 20

**Review by michalBassa — APPROVED** [review]

**Pull request merged**